src/test/java/com/ksm/wstbackoffice/controller/AddressControllerIT.java

```diff
@@ -19,16 +24,32 @@
         @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/addressDelete.sql")
 })
 public class AddressControllerIT extends BaseControllerIT {
-    @Test
-    public void findByIdTest() {
-        given().
-            pathParam("id", 1).
-        when().
-            get("addresses/{id}").
-        then().
-            statusCode(200).
-            body("id", equalTo(1)).
-            body("country.iso3166", equalTo("804"));
+    @ParameterizedTest
+    @CsvSource({
+            ",",
+            "0"
+    })
+    public void findByIdinvalidValuesAreNotAllowedTest(Long id) {
+        Throwable thrown = catchThrowable(() -> findByIdTest(id));
+        assertThat(thrown)
+                .isInstanceOf(CheckValueException.class)
+                .hasMessageContaining("id is mandatory");
+    }
+
+    @ParameterizedTest
+    @CsvSource({"1"})
+    public void findByIdTest(Long id) {
+        try {
+            given().
+                pathParam("id", id).
+            when().
+                get("addresses/{id}").
+            then().
+                statusCode(200).
+                body("id", equalTo(id.intValue()));
+        } catch (Exception ex) {
```

Comment:
This is not a good approach to catch an exception when doing HTTP requests.
The general rules are:
1. return 404 (NOT FOUND) if the entity is not found by id
2. return 400 (BAD REQUEST) if the provided id is not valid
So your controllers should fit these rules.
And here you should provide different values as id and expecting different status codes as responses.